apps/web-portal-client/lib/utils/BrowserStorage.mjs

```diff
@@ -0,0 +1,30 @@
+export class BrowserStorage {
```

Comment:
Let's try a more storage independent abstractions

```js
export class AppPersistentStorage {

   constructor (){
      this.#local = {} 
      this.#session = {} // probably make sense to add it a later on when we need it
   }

   writeApiToken()
   cleanApiToken()

    writeLastRoutePath()
    cleanLastRoutePath()
   
   
   

}


export new AppPersistantStorage()
```

So other parts of app don't see keys and type of persistent storage

Reply:
done